### PR TITLE
Paralyzer

### PR DESCRIPTION
**Pull request by dyusuf**

### CODE REVIEW

File: tools/rna_tools/paralyzer/paralyzer.xml [hunk 1]
```diff
@@ -0,0 +1,380 @@
+<tool id="paralyzer" name="PARalyzer" version="0.1.0">
```
Comment (bgruening) [inline tools/rna_tools/paralyzer/paralyzer.xml]: Maybe version 1.5?

File: tools/rna_tools/paralyzer/paralyzer.xml [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+        faToTwoBit '$refGenomeSource.ownFile' ownFile.2bit &&
+        echo 'GENOME_2BIT_FILE=ownFile.2bit' > input.ini &&
+    #else
+        echo 'GENOME_2BIT_FILE=$refGenomeSource.builtin.fields.path' > input.ini &&
+    #end if
+
+    echo 'SAM_FILE=$input_sam$collapse' >> input.ini &&
+
+    #if $methods.choice == "ADDITIONAL_NUCLEOTIDES_BEYOND_SIGNAL":
+      echo 'ADDITIONAL_NUCLEOTIDES_BEYOND_SIGNAL=$methods.max_num' >> input.ini &&
```
Comment (bgruening) [inline tools/rna_tools/paralyzer/paralyzer.xml]: Do you know the config file feature of Galaxy? You should really use this, it is so much simpler and cleaner.
Look at this one: https://github.com/galaxyproject/tools-iuc/blob/596a23abc0c657eff218d16df685b427ae7cdd0a/tools/vegan/vegan_fisher_alpha/vegan_fisher_alpha.xml#L16

File: tools/rna_tools/paralyzer/paralyzer.xml (hunk 1 above)
Comment (dyusuf) [inline tools/rna_tools/paralyzer/paralyzer.xml]: yeah, I thought I made sure that you could not make a comment on this one :D

File: tools/rna_tools/paralyzer/paralyzer.xml [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+
+    echo 'OUTPUT_DISTRIBUTIONS_FILE=out.distribution' >> input.ini &&
+    echo 'OUTPUT_GROUPS_FILE=out.groups' >> input.ini &&
+    echo 'OUTPUT_CLUSTERS_FILE=out.clusters' >> input.ini &&
+
+    ## execute paralyzer
+    ## note the 2nd argument is the memory parameter
+    ## the parameter can be override in job_conf.xml e.g.
+    ## <env id="_JAVA_OPTIONS">-Xmx12G</env>
+    PARalyzer 2G input.ini
```
Comment (dyusuf) [inline tools/rna_tools/paralyzer/paralyzer.xml]: cool

**Comment by bgruening** [comment]
Nice, let's wait for the tests! Thanks Dili!

**Comment by dyusuf** [comment]
@bgruening thank you! learned a lot. always new stuff to learn with galaxy

**Comment by bgruening** [comment]
Great stuff, thanks!

**Comment by dyusuf** [comment]
@bgruening I have not found it at https://toolshed.g2.bx.psu.edu/

**Pull request merged**